src/material/card/_card-theme.scss

```diff
@@ -1,70 +1,80 @@
-      // use them because they output under a `.mdc-card` selector whereas the rest of the theme
-      // isn't under any selector. Even if the mixin is pulled out of the selector, it throws a
-      // different error.
-      @include private.private-theme-elevation(1, $config);
-      @include mdc-elevated-card-theme.theme((
-        container-color: $surface,
-      ));
-    }
+    // TODO(mmalerba): This should be based on tokens, but the elevation tokens don't seem to be
+    //  working currently. They wind up setting values like `box-shadow: var(--e); --e: 1`
+    @include private.private-theme-elevation(1, $config);
+  }
 
-    .mat-mdc-card-outlined {
-      @include private.private-theme-elevation(0, $config);
-      @include mdc-outlined-card-theme.theme((
-        outline-color: if(
-          meta.type-of($on-surface) == color and meta.type-of($surface) == color,
-          color.mix($on-surface, $surface, 12%),
-          $on-surface
-        )
-      ));
-    }
+  .mat-mdc-card-outlined {
+    // TODO(mmalerba): This should be based on tokens, but the elevation tokens don't seem to be
+    //  working currently. They wind up setting values like `box-shadow: var(--e); --e: 1`
+    @include private.private-theme-elevation(0, $config);
+  }
 
-    // Card subtitles are an Angular Material construct (not MDC), so we explicitly set their
-    // color to secondary text here.
-    .mat-mdc-card-subtitle {
-      color: theming.get-color-from-palette($foreground, secondary-text);
-    }
+  // Card subtitles are an Angular Material construct (not part of MDC),
+  // so we add their color styles here.
+  // TODO(mmalerba): Add custom tokens for this.
+  .mat-mdc-card-subtitle {
+    color: theming.get-color-from-palette(map.get($config, foreground), secondary-text);
   }
 }
 
 @mixin typography($config-or-theme) {
   $config: typography.private-typography-to-2018-config(
-      theming.get-typography-config($config-or-theme));
-  @include mdc-helpers.using-mdc-typography($config) {
-    // Card subtitles and titles are an Angular Material construct (not MDC), so we explicitly
-    // set their typographic styles here.
-    .mat-mdc-card-title {
-      @include mdc-typography.typography(headline6);
-    }
+          theming.get-typography-config($config-or-theme));
+  $mdc-elevated-card-typography-tokens: m2-mdc-elevated-card.get-typography-tokens($config);
+  $mdc-outlined-card-typography-tokens: m2-mdc-outlined-card.get-typography-tokens($config);
+
+  // Add values for MDC card tokens.
+  .mat-mdc-card {
+    @include mdc-elevated-card-theme.theme($mdc-elevated-card-typography-tokens);
+    @include mdc-outlined-card-theme.theme($mdc-outlined-card-typography-tokens);
+
+    // Card subtitles and titles are an Angular Material construct (not part of MDC),
+    // so we explicitly set their typographic styles here.
+    // TODO(mmalerba): Add custom tokens for this.
```

Comment:
Would it make sense to do these now? My experience with introducing tokens later (see https://github.com/angular/components/pull/26197) was that it's harder since they can change the specificity.

Reply:
I suppose, some of them I was purposely waiting on because I wanted to see about getting new tokens added in MDC, but this one seems like a clear-cut case of just needing to make our own because MDC doesn't even have a concept of card titles and subtitles. I'll work on adding them

Reply:
PTAL at the last commit to see my approach to adding custom tokens